functional_tests/test_db_online_offline.py

```diff
@@ -461,34 +453,30 @@ def test_online_to_offline_changes_feed_controlled_close_longpoll(cluster, conf,
         futures[executor.submit(admin.take_db_offline, "db")] = "db_offline_task"
 
         for future in concurrent.futures.as_completed(futures):
-            try:
-                task_name = futures[future]
-
-                if task_name == "db_offline_task":
-                    log.info("DB OFFLINE")
-                    # make sure db_offline returns 200
-                    assert(future.result() == 200)
-                if task_name == "docs_push":
-                    log.info("DONE PUSHING DOCS")
-                    doc_add_errors = future.result()
-                if task_name == "polling":
-                    # Long poll will exit with 503, return docs in the exception
-                    log.info("POLLING DONE")
-                    try:
-                        docs_in_changes = future.result()
-                    except Exception as e:
-                        log.info(e)
-                        log.info("POLLING DONE EXCEPTION")
-                        log.info("AARGS: {}".format(e.args))
-                        docs_in_changes = e.args[0]["docs"]
-                        last_seq_num = e.args[0]["last_seq_num"]
-                        log.info("DOCS FROM longpoll")
-                        for k, v in docs_in_changes.items():
-                            log.info("DFC -> {}:{}".format(k, v))
-                        log.info("LAST_SEQ_NUM FROM longpoll {}".format(last_seq_num))
-
-            except Exception as e:
-                print("Futures: error: {}".format(e))
+            task_name = futures[future]
+
+            if task_name == "db_offline_task":
+                log.info("DB OFFLINE")
+                # make sure db_offline returns 200
+                assert(future.result() == 200)
+            if task_name == "docs_push":
+                log.info("DONE PUSHING DOCS")
+                doc_add_errors = future.result()
+            if task_name == "polling":
+                # Long poll will exit with 503, return docs in the exception
+                log.info("POLLING DONE")
+                try:
+                    docs_in_changes = future.result()
+                except Exception as e:
+                    log.info(e)
```

Comment:
In this test case, the exception is expected, so I am going with warn, because it is not technically an error